packages/db/src/db.generated.d.ts

```diff
@@ -192,8 +192,12 @@ export interface Offline {
 export interface OwUser {
   auth0Sub: string | null
   createdAt: Generated<Timestamp>
+  email: string
   id: Generated<string>
+  lastSyncedAt: Timestamp | null
```

Comment:
```suggestion
  lastSyncedAt: Timestamp
```
This should just be default now?

Reply:
Yea agree, good suggestion.